e2etest/pkg/metallb/config.go

```diff
@@ -74,3 +78,22 @@ func WithRouterID(peers []metallbv1beta2.BGPPeer, routerID string) []metallbv1be
 	}
 	return peers
 }
+
+func BGPPeerSecretReferences(containers []*frrcontainer.FRR) map[string]corev1.Secret {
```

Comment:
This can return a slice, I think?

Reply:
I think this can be still be a map to align with [PasswordSecrets](https://github.com/metallb/metallb/blob/1e364f511ba9e29fb0ecbfe395392576376a82db/internal/config/config.go#L43) in ClusterResources, I chose map over slice as its mainly used for a secret lookup for secret ref in bgp peer.

Reply:
You are right.